libjlc/include/jlc/command.hpp

```diff
@@ -63,12 +65,16 @@ class prscmd final : public command {
 		const bool & verbose,
 		const bool & rdynamic,
 		const bool & suppress,
+		const bool & pthread,
 		const standard & std)
 	{
-		std::unique_ptr<prscmd> cmd(new prscmd(ifile, Ipaths, Dmacros, Wwarnings, flags, verbose, rdynamic, suppress, std));
+		std::unique_ptr<prscmd> cmd(new prscmd(ifile, Ipaths, Dmacros, Wwarnings, flags, verbose, rdynamic, suppress, pthread, std));
 		return passgraph_node::create(pgraph, std::move(cmd));
 	}
 
+	static std::string
```

Comment:
What is this function supposed to do?

Reply:
Right, I just saw it further down.....

Reply:
Could you make this method private, please. It is only internally used in the prscmd class.